src/syscheckd/db/src/fimDBUtils.hpp

```diff
@@ -21,16 +44,18 @@ namespace FimDBUtils
      * @return a vector with paths asociated to the inode.
      */
 
+    template <typename T>
     static std::vector<std::string> getPathsFromINode(const unsigned long inode, const unsigned long dev)
     {
         std::vector<std::string> paths;
         nlohmann::json resultQuery;
 
         try
         {
-            const auto filter { std::string("WHERE inode=") + std::to_string(inode) + std::string(" AND dev=") + std::to_string(dev) };
-            const auto query = FIMDBHelper::dbQuery(FIMBD_FILE_TABLE_NAME, FILE_PRIMARY_KEY, filter, FILE_PRIMARY_KEY);
-            FIMDBHelper::getDBItem<FIMDB>(resultQuery, query);
+            const auto filter = std::string("WHERE inode=") + std::to_string(inode) + std::string(" AND dev=") + std::to_string(dev);
```

Comment:
I think you can get rid of the construction of strings objects for "WHERE inode=" and " AND dev=" as you are already creating strings objects with the std::to_string() for concatenation purposes.

Reply:
You are right, I'm changing that ! Thank you !